VAMobile/src/utils/secureMessaging.tsx

```diff
@@ -537,90 +529,95 @@ export const getLinkifiedText = (
     const url2Match = URL2_REGEX_EXP.exec(text)
     if (emailMatch) {
       //matches <email address> only
-      textReconstructedBody.push(
-        <TouchableWithoutFeedback
-          onPress={() => {
-            launchExternalLink('mailto:' + text)
-          }}
-          accessibilityRole="link"
-          accessible={true}
-          accessibilityLabel={text}
-          accessibilityHint={t('openInEmailMessaging.a11yHint')}>
-          <TextView variant="MobileBodyLink">{text}</TextView>
-        </TouchableWithoutFeedback>,
+      const iconProps: IconProps = {
+        name: 'ExternalLink',
+      }
+      const linkProps: LinkProps = {
+        text: text,
+        type: 'custom',
```

Comment:
Can't this use the 'url' type since it's a URL?

Reply:
good call, updated